addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/java/org/nuxeo/template/serializer/service/SerializerServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package org.nuxeo.template.serializer.service;
+
+import org.nuxeo.runtime.model.ComponentInstance;
+import org.nuxeo.runtime.model.DefaultComponent;
+import org.nuxeo.template.serializer.executors.Serializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SerializerServiceImpl extends DefaultComponent implements SerializerService {
+
+    private static final String SERIALIZER_EP_NAME = "serializers";
+    private static final String DEFAULT_SERIALIZER_NAME = "default";
+
+    private Map<String, SerializerContribution> serializers = new HashMap<>();
+
+    @Override
+    public void registerContribution(Object contribution, String extensionPoint, ComponentInstance contributor) {
+        if (SERIALIZER_EP_NAME.equals(extensionPoint)) {
+            SerializerContribution sc = (SerializerContribution) contribution;
+            serializers.put(sc.getName(), sc);
+            return;
+        }
+        throw new UnknownExtensionPoint();
+    }
+
+    @Override
+    public void unregisterContribution(Object contribution, String extensionPoint, ComponentInstance contributor) {
+        if (SERIALIZER_EP_NAME.equals(extensionPoint)) {
+            SerializerContribution sc = (SerializerContribution) contribution;
+            serializers.remove(sc.getName());
+            return;
+        }
+        throw new UnknownExtensionPoint();
+    }
+
+    @Override
+    public Serializer getSerializer(String name) {
+        SerializerContribution contrib = serializers.get(name);
+
+        if (contrib == null) {
+            contrib = serializers.get(DEFAULT_SERIALIZER_NAME);
+            if (contrib == null) {
+                throw new UnknownSerializer(name);
```

Comment:
Could you use a `NuxeoException` instead?

Reply:
Ok I extends NuxeoException. I don't like to have generic exception. If you are agree...

Reply:
I extended NuxeoExcpetion. => done

Reply:
If you are agree I prefer to keep my specific exception and extends it as NuxeoException... What do you think ? I did the modification in this way...